docs/physics/spectrum/basic.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>"can also be gotten" --&gt; "can also be retrieved"?</p><p>Also, I would delete "(which is included if you wish to remove the comments and run that cell)."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>